### DISCUSSION TITLE
What is ":resources" in /usr/lib/firetools/uiapps?

### DISCUSSION THREAD

**Discussion by Rosika2** [netblue30/firejail#6055] [Q&A] [answered]
Hi all, :wave: 

as I´m using the latest version of `firetools` now (0.9.72) I have to edit `~/.config/firetools/uiapps` for adding my personal entries.
That´s fine and I´m getting along with it quite well, it seems.

I still have a question regarding the **syntax** of the entries.

In `/usr/lib/firetools/uiapps ` it says:

> Each line defines an application, with semicolons separated fields as follows:
> 
>  executable; description; icon; (optional) command for starting Firejail

O.K., I get that.

Here´s an example I understand:

`thunderbird;Thunderbird Email Client;thunderbird`

But what about this one:

`firefox;Mozilla Firefox;:resources/firefox`  :question: 

The third field should be for the icon, it says. I don´t understand the ":resources" bit. There are only few entries that use this nomenclature.

Many thanks in advance for your help.
Cheers from Rosika :slightly_smiling_face:

**Comment by ghost** [comment, marked as answer]
> firefox;Mozilla Firefox;:resources/firefox ❓
The third field should be for the icon, it says. I don´t understand the ":resources" bit. There are only few entries that use this nomenclature.

The referenced `resources` is a subdirectory of the firetools repository where the png icon is stored:

https://github.com/netblue30/firetools/tree/master/src/firetools/resources

**Comment by Rosika2** [comment]
Hi @glitsj16, :wave: 

thank you very much for your reply. 

O.K.,  a "subdirectory of the firetools repository" means that the `png` file doesn´t reside on the local system then. 
For firefox it´s this url, as I found out (thanks to your link):

https://github.com/netblue30/firetools/blob/master/src/firetools/resources/firetools.png

Thank you very much for enlightening me. :heart: 

Many greetings from Rosika :slightly_smiling_face:
